POS/Operation/02_SelectOffer.md

```diff
@@ -87,34 +155,116 @@ Offers for POS are created, see [Manage offers for POS](../Integration/07_Manage
+
+#### Procedure
+
+*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 
+
+![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")
+
+1. Click the *Find offer by nam...* search bar in the upper left part of the tab.  
+    The cursor is displayed in the search bar.
+
+2. Enter the name or the SKU of the voucher you are searching for in the search bar.
+    The offers that match the search are displayed in a list below the search bar.
+
+    > [Info] The search already provides you with proposals as you type.
+
+3. Select the desired voucher by clicking the corresponding offer in the list.   
+    The *Voucher details* window is displayed.
+
+    ![Voucher details](../../Assets/Screenshots/POS/Sales/Cashpoint/VoucherDetails.png "[Voucher details]")
+
+4. Scan the voucher code with the barcode scanner or enter the code manually.     
+    If the barcode was successfully scanned, the voucher code is automatically added to the *Voucher code* field. The cursor is placed in the *Amount* field, where a default value is provided, which you can overwrite.
+
+5. Enter the amount in the *Amount* field. Note that this amount will be added to any amount already on the voucher.
+
+6. Click the [SAVE] button.   
+    The voucher with its newly created amount is added to the bill list.
 
-If the same offer is selected more than once, it is displayed in the bill list as one product with the corresponding quantity indication. By using the single scan feature, each offer is listed as a single product in the bill list, even if it is the same offer. This allows you to treat each product in the booking individually, for instance to apply an individual discount.
+
+<!--- NEU-->
+## Verify the age
+
+Some offers are not suitable for children and young people, for example offers such as alcoholic beverages, cigarettes, or energy drinks. When you select an offer, Actindo checks whether an age verification is necessary before the offer is added to the bill list.
 
 #### Prerequisites
 
-Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- An age restriction for the offer is defined, for example 18 or 16.    
+- The *Display age rating* setting is activated. For detailed information, see [Display age rating](../UserInterface/02a_GlobalSettings.md#display-age-rating).
 
 #### Procedure
 
-*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*
+*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT* 
 
 ![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Cashpoint.png "[Cashpoint]")
 
-1. Activate the *Single scan* toggle in the bill list on the right side of the pay desk interface.   
+1. Select an offer.   
+    If the offer requires an age verification, the *Age verification* window is displayed.
```

Comment:
```suggestion
    The *Age verification* window is displayed.
```
Wir haben oben gesagt, dass Age verification is eingestellt.

Reply:
s.o.